src/SriPlugin.ts

```diff
@@ -26,6 +26,7 @@ export default class SriPlugin implements webpack.WebpackPluginInstance {
           }
 
           filePath = filePath.replace(/\?id=\w{20}/, '')
+          filePath = filePath.replace(/\?\w{32}/, '')
```

Comment:
This will break the Laravel Mix cache busting feature as it won't remove the `?id=` part.

Something like this would be better:
```suggestion
          filePath = filePath.replace(/\?(\w{32}|id=\w{20})/, '')
```

Reply:
It will still replace the id= part too, the regexes are just on two separate lines now. Your solution is cleaner though.